llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -862,7 +864,8 @@ class VPRecipeBase : public ilist_node_with_parent<VPRecipeBase, VPBasicBlock>,
 protected:
   /// Compute the cost of this recipe using the legacy cost model and the
   /// underlying instructions.
-  InstructionCost computeCost(ElementCount VF, VPCostContext &Ctx) const;
+  virtual InstructionCost computeCost(ElementCount VF,
+                                      VPCostContext &Ctx) const;
```

Comment:
nit: the comment can be refined as the legacy cost model can be overriden.

Reply:
Adjusted the comment, thanks!